development/circular-deps.jsonc

```diff
@@ -241,88 +222,18 @@
     "ui/store/actions.ts"
   ],
   [
-    "ui/ducks/domains.js",
-    "ui/ducks/index.js",
-    "ui/store/actions.ts",
-    "ui/store/store.ts"
-  ],
-  [
-    "ui/ducks/index.js",
-    "ui/ducks/metamask/metamask.js",
-    "ui/ducks/ramps/ramps.ts",
-    "ui/pages/confirmations/confirmation/ResultTemplate.ts",
-    "ui/pages/confirmations/confirmation/templates/error.js",
-    "ui/pages/confirmations/confirmation/templates/index.js",
-    "ui/selectors/multichain.ts",
-    "ui/selectors/selectors.js",
-    "ui/store/actions.ts",
-    "ui/store/store.ts"
-  ],
-  [
-    "ui/ducks/index.js",
-    "ui/ducks/metamask/metamask.js",
-    "ui/ducks/swaps/swaps.js",
-    "ui/hooks/useTokenFiatAmount.js",
-    "ui/pages/confirmations/confirmation/ResultTemplate.ts",
-    "ui/pages/confirmations/confirmation/templates/error.js",
-    "ui/pages/confirmations/confirmation/templates/index.js",
-    "ui/selectors/selectors.js",
-    "ui/store/actions.ts",
-    "ui/store/store.ts"
-  ],
-  [
-    "ui/ducks/index.js",
     "ui/ducks/metamask/metamask.js",
-    "ui/ducks/swaps/swaps.js",
     "ui/pages/confirmations/confirmation/ResultTemplate.ts",
     "ui/pages/confirmations/confirmation/templates/error.js",
     "ui/pages/confirmations/confirmation/templates/index.js",
     "ui/selectors/selectors.js",
-    "ui/store/actions.ts",
-    "ui/store/store.ts"
+    "ui/store/actions.ts"
   ],
   [
-    "ui/ducks/index.js",
     "ui/ducks/metamask/metamask.js",
-    "ui/pages/confirmations/confirmation/ResultTemplate.ts",
-    "ui/pages/confirmations/confirmation/templates/error.js",
-    "ui/pages/confirmations/confirmation/templates/index.js",
-    "ui/selectors/selectors.js",
-    "ui/store/actions.ts",
-    "ui/store/store.ts"
-  ],
-  [
-    "ui/ducks/index.js",
-    "ui/ducks/ramps/ramps.ts",
-    "ui/pages/confirmations/confirmation/ResultTemplate.ts",
-    "ui/pages/confirmations/confirmation/templates/error.js",
-    "ui/pages/confirmations/confirmation/templates/index.js",
     "ui/selectors/multichain.ts",
-    "ui/selectors/selectors.js",
-    "ui/store/actions.ts",
-    "ui/store/store.ts"
-  ],
-  [
-    "ui/ducks/index.js",
-    "ui/ducks/swaps/swaps.js",
-    "ui/helpers/utils/token-util.js",
-    "ui/hooks/useTokenFiatAmount.js",
-    "ui/store/actions.ts",
-    "ui/store/store.ts"
-  ],
-  [
-    "ui/ducks/index.js",
-    "ui/ducks/swaps/swaps.js",
-    "ui/store/actions.ts",
-    "ui/store/store.ts"
-  ],
-  [
-    "ui/ducks/metamask/metamask.js",
-    "ui/pages/confirmations/confirmation/ResultTemplate.ts",
-    "ui/pages/confirmations/confirmation/templates/error.js",
-    "ui/pages/confirmations/confirmation/templates/index.js",
-    "ui/selectors/selectors.js",
-    "ui/store/actions.ts"
+    "ui/selectors/selectors.js"
   ],
-  ["ui/ducks/metamask/metamask.js", "ui/selectors/selectors.js"]
+  ["ui/ducks/metamask/metamask.js", "ui/selectors/selectors.js"],
+  ["ui/selectors/multichain.ts", "ui/selectors/selectors.js"]
```

Comment:
Good work reducing circular dependencies, but this is the second of two _**new**_ circular dependencies that we cannot allow.

Reply:
Oh, you're right.  It was an existing longer dependency chain, and you shortened it.